tests/test_client.py

```diff
@@ -43,6 +41,7 @@
 
 
 mpr = MPRester()
+print(os.getenv("MP_API_KEY"), mpr.api_key, mpr.session.headers)
```

Comment:
Thanks for reviewing! 

I don't really think this would work because secrets are not passed to the environment when triggered from a fork. Honestly I don't know any way around this except for running test locally, or wait until this PR get merge and run CI from there.